command/title_body_survey.go

```diff
@@ -145,21 +181,150 @@ func titleBodySurvey(cmd *cobra.Command, providedTitle, providedBody string, def
 		qs = append(qs, bodyQuestion)
 	}
 
-	err := SurveyAsk(qs, &inProgress)
+	err := SurveyAsk(qs, data)
 	if err != nil {
-		return nil, fmt.Errorf("could not prompt: %w", err)
+		return fmt.Errorf("could not prompt: %w", err)
 	}
 
-	if inProgress.Body == "" {
-		inProgress.Body = templateContents
+	if data.Body == "" {
+		data.Body = templateContents
 	}
 
-	confirmA, err := confirmSubmission(allowPreview)
+	confirmA, err := confirmSubmission(!data.HasMetadata(), true)
```

Comment:
The args here make it a little confusing. Maybe pull them out as individual vars instead of passing the args inline?

Reply:
Good call!